optional_plugins/varianter_cit/avocado_varianter_cit/Cit.py

```diff
@@ -98,8 +93,7 @@ def find_better_solution(self, counter, matrix):
         :return: new matrix and is changes have been successful?
         """
         while self.combination_matrix.total_uncovered != 0:
-            if self.debug:
-                print_progress(counter)
+            LOG.info(self.__throbber.render(), extra={"skip_newline": True})
```

Comment:
This should probably become `LOG.debug()`, given that this happens when this is set to info:

```
$ avocado variants --cit-order-of-combinations=1 --cit-parameter-file examples/varianter_cit/params.ini 
/CIT Variants (5):
Variant gold-triangle-gas-plastic-cathodic-3-4-4-1:    /
Variant black-circle-solid-leather-cathodic-7-1-2-2:    /
Variant green-square-liquid-aluminum-cathodic-6-5-5-4:    /
Variant red-circle-solid-plastic-anodic-1-2-1-3:    /
Variant black-triangle-solid-leather-anodic-5-3-3-5:    /
```

Notice the second line.  IMO, a different output should only be produced when `--debug` is given.

Reply:
There is the `LOG.info()` because I think that there should be some information about computing. With a bigger size of parameters, the computing time could be about one hour so the user should have some visual information about running of the application.